src/pecha_uploader/index/extract.py

```
@@ -14,6 +14,17 @@ def get_index(index: str):
     url = f"{index_url}/{prepare_index_str}?with_content_counts=1"
     req = urllib.request.Request(url, method="GET", headers=headers)
     try:
-        response = urllib.request.urlopen(req)  # noqa
+        response = urllib.request.urlopen(req)  # noqa]
+        return {"status": True}
     except HTTPError as e:
-        print("Error code: ", e.code)
+        # Handle HTTP errors
+        log_error(
+            text_error_logger,
+            f"HTTP Error {e.code} occurred extracting index: {e.read().decode('utf-8')}",
+        )
+        return {"status": False}
```

dont return status like this. If exception occurs, it should raise a exception.